packages/nuka/src/carousel.tsx

```diff
@@ -175,11 +186,11 @@ export const Carousel = React.forwardRef<HTMLDivElement, CarouselProps>(
     }, []);
 
     const forwardedRef = useForwardRef<HTMLDivElement>(ref);
-    const carouselRef = forwardedRef || defaultCarouselRef;
+    const carouselDivRef = forwardedRef || defaultCarouselDivRef;
```

Comment:
I renamed the ref pointing to the carousel's div to `carouselDivRef` to avoid confusing with the new `carouselRef` prop, which is now the external carousel ref.

I kept the forwarded ref as the ref to the div to maintain backwards compatibility.